config/routes.rb

```diff
@@ -0,0 +1,13 @@
+Rails.application.routes.draw do
+  #get 'root/index'
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  # root "articles#index"
+  root 'root#index'
+  namespace :api do
+    namespace :v1 do
+      resources :greetings, only: [:index] 
+    end
+  end
```

Comment:
- [ ] Kindly make sure your route was created correctly as this shows an error when your app is started

![Screenshot_505](https://user-images.githubusercontent.com/69638013/197365028-d547ef77-b166-469b-aecf-2a547cdcc564.png)